src/main/java/org/akip/repository/TaskDefinitionRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.akip.repository;
+
+import org.akip.domain.TaskDefinition;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface TaskDefinitionRepository extends JpaRepository<TaskDefinition, Long> {
+
+    public Optional<TaskDefinition> findTaskDefinitionByBpmnProcessDefinitionIdAndTaskId(String bpmnProcessDefiinitionId, String taskId);
```

Comment:
Rename method to `findByBpmnProcessDefinitionIdAndTaskId`

Reply:
Done!